.eslintrc.cjs

```diff
@@ -150,6 +151,7 @@ module.exports = {
       'error',
       {
         cases: {
+          camelCase: true,
```

Comment:
i'm not sure exactly why the robot added this

Reply:
yeah we need to remove this. it's supposed to enforce vue files only i think